addon/-private/system/model/model.js

```diff
@@ -1125,11 +1123,26 @@ const Model = Ember.Object.extend(Ember.Evented, {
  @type {Object}
  */
 Object.defineProperty(Model.prototype, 'data', {
+  configurable: false,
   get() {
     return this._internalModel._data;
   }
 });
 
+Object.defineProperty(Model.prototype, 'id', {
+  configurable: false,
+  set(id) {
+    this._internalModel.setId(id);
+  },
```

Comment:
I believe we need to add `this.notifyPropertyChange('id')` after the call to set, because by using our own getter/setter observers and computeds are no longer able to subscribe via installed getters/setters.